### DISCUSSION TITLE
stopped with error code 1

### DISCUSSION THREAD

**Discussion by pere000** [neutralinojs/neutralinojs#767] [General]
Installed Jan, 7, 2022
npm i -g @neutralinojs/neu

:~/hello-world$ neu run
Starting process: neutralino-linux_x64  --load-dir-res --path=. --export-auth-info --neu-dev-auto-reload
neutralino-linux_x64 was stopped with error code 1

**Comment by shalithasuranga** [comment]
Hey.. thanks for reporting the issue. Could you please run `./neutralino-linux_x64 --load-dir-res --path=. --export-auth-info --neu-dev-auto-reload` manually and check the output. FYI: we will release a new version of CLI with `neu run --verbose` support to find initialization issues like these :)
